internal/services/paloalto/schema/plan_data.go

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package schema
+
+import (
+	"github.com/hashicorp/go-azure-helpers/lang/pointer"
+	"github.com/hashicorp/go-azure-sdk/resource-manager/paloaltonetworks/2023-09-01/firewalls"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/tf/pluginsdk"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/tf/validation"
+)
+
+type PlanData struct {
+	BillingCycle string `tfschema:"billing_cycle"`
+	PlanId       string `tfschema:"plan_id"`
+	UsageType    string `tfschema:"usage_type"`
+}
+
+func PlanDataSchema() *pluginsdk.Schema {
+	return &pluginsdk.Schema{
+		Type:     pluginsdk.TypeList,
+		Optional: true,
+		MaxItems: 1,
+		Elem: &pluginsdk.Resource{
+			Schema: map[string]*pluginsdk.Schema{
+				"plan_id": {
+					Type:         pluginsdk.TypeString,
+					Optional:     true,
+					ForceNew:     true,
+					Default:      "panw-cloud-ngfw-payg",
+					ValidateFunc: validation.StringIsNotEmpty,
+				},
+
+				"billing_cycle": {
+					Type:         pluginsdk.TypeString,
+					Optional:     true,
+					ForceNew:     true,
+					Default:      firewalls.BillingCycleMONTHLY,
+					ValidateFunc: validation.StringInSlice(firewalls.PossibleValuesForBillingCycle(), false),
+				},
+
+				"usage_type": {
+					Type:         pluginsdk.TypeString,
+					Optional:     true,
+					ForceNew:     true,
+					Default:      firewalls.UsageTypePAYG,
```

Comment:
Could you double check if it has the default value since [Swagger](https://github.com/Azure/azure-rest-api-specs/blob/58740206b853320974ef5e4864f7be8120b15a27/specification/paloaltonetworks/resource-manager/PaloAltoNetworks.Cloudngfw/stable/2023-09-01/PaloAltoNetworks.Cloudngfw.json#L6220) indicates it's not a required property? Thanks.